### PR TITLE
missed a damn dash

### PR DESCRIPTION
**Pull request by agubler**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ install:
 - cd $PROJECT_DIR
 - travis_retry npm install grunt-cli
 - travis_retry npm install
-  cd ..
+- cd ..
```
Comment (uttrasey) [inline .travis.yml]: http://giphy.com/gifs/thelovechild-3qCAEJQdh2MWn5azQY

Comment (uttrasey) [inline .travis.yml]: giphy

Comment (Tomdye) [inline .travis.yml]: hahaha

Comment (uttrasey) [inline .travis.yml]: @agubler don't worry about the missing '-' just please don't let it happen again

**Pull request merged**